app/commands/git/sync_pull_requests.rb

```diff
@@ -0,0 +1,163 @@
+module Git
+  class SyncPullRequests
+    include Mandate
+
+    def call
+      repos.each do |repo|
+        @repo = repo
```

Comment:
Not a big fan of changing the instance variable in the loop like this. I think I'd just have a seperate class for `SyncPullRequestsFromRepo.(repo)` and move everything else into that. That'll make testing easier too.